internal/daemon/hostsidemanager.go

```diff
@@ -268,9 +267,7 @@ func (d *HostSideManager) Serve(listener net.Listener) error {
 
 func (d *HostSideManager) Stop() {
```

Comment:
General question, who do who expect to invoke a sideManager's stop() method? Makes sense for the daemon to clean up after itself, but trying to understand when in the sequence this would be called.

Reply:
Stop() is callable if you don't use ListenAndServe. It only makes sense to invokte stop if Serve is in a go-routine. If we have multiple side managers, if one errors out, we want to error out the whole binary. For that, we will call stop on all the other SideManagers that have not errored out to stop all the go routines in which we run Serve.